src/rejson.c

```diff
@@ -1917,7 +1917,7 @@ int Module_CreateCommands(RedisModuleCtx *ctx) {
         return REDISMODULE_ERR;
 
     if (RedisModule_CreateCommand(ctx, "json.debug", JSONDebug_RedisCommand, "readonly getkeys-api",
-                                  1, 1, 1) == REDISMODULE_ERR)
+                                  2, 2, 1) == REDISMODULE_ERR)
```

Comment:
I'm not sure that this is the right fix it.

I think the problem is at https://github.com/RedisJSON/RedisJSON/blob/1.0/src/rejson.c#L258, which should become:

```C
        RedisModuleKey *key = RedisModule_OpenKey(ctx, argv[3], REDISMODULE_READ);
```